

<launch>  
  
  <include file="$(find hsrb_moveit_config)/launch/hsrb_demo_with_controller.launch"> 
  <arg name="use_gui" value="true"/> 
  </include>


  <param name="file_name" value="/known_locations_storing.yaml"/>
  <node pkg="hmm_navigation"  name="pumas_nav" type="pumas_nav.py" output="screen" />
  <node pkg="known_locations_tf_server"  name="known_locs" type="known_locs_node.py" output="screen" />
  <node pkg="object_classification"  name="objs_reco" type="classification_server.py"  />  
  <node pkg="segmentation"  name="segmentation" type="segment_node.py"  />  
  <node pkg="segmentation"  name="placing_finder" type="placing_finder_node.py"  />  
  <node pkg="smach_viewer"  name="smach_viewer" type="smach_viewer.py"  />    
  <node pkg="point_cloud"  name="point_cloud_filter_node" type="passthrough_filter"/>  
  <node pkg="action_server"  name="action_server" type="ACT_SRVR_grasp_mio.py"/>  
  <node pkg="action_server"  name="place_action_server" type="ACT_SRVR_place.py"/>   
  
</launch>  